packages/aws-cdk/lib/api/aws-auth/credentials.ts

```diff
@@ -6,7 +6,6 @@ export enum Mode {
 }
 
 /**
- * @experimental
```

Comment:
@rix0rrr @NetaNir which APIs exactly are we exporting from the CDK CLI? Are we really ready to call these stable? I am concerned that this will signal to folks that they can use the CLI as a library and will bound is to this APIs... I'd hold off with removing the @experimental tags here (the CLI is also not part of the aws-cdk-lib, so I don't think this is required for v2)

Reply:
@eladb But it is a part of the AWS CDK, and I don't think there will be any notable distinction in our v2 announcements between the CLI and the framework, at least not in a way that will be easy for customers understand. If we break a public API in the CLI post GA, it will most likely be perceived as not keeping our stability promise. In all honesty, I don't think this annotation was ever understood/noticed by customers.

Reply:
@eladb - we should just remove all exports from the package [index](https://github.com/aws/aws-cdk/blob/master/packages/aws-cdk/lib/index.ts) so that this package cannot be used as a library.
(need to confirm if npm/node allows this)

That feels like the correct solution here, no?